requirements.txt

```
@@ -0,0 +1,8 @@
+# used in sphinx documentation build
+conda-package-streaming>=0.5.0
```

```suggestion
conda-package-streaming>=0.6.0
```

I tested this locally and had breaking tests until I bumped the version here. Does it even make sense having this file here? (I saw you did update it in `setup.py`) Could we just add the essential dependencies to the `setup.py` file and instead just have `requirements-dev.txt` with all the dependencies necessary for running tests and building documentation? 

Also, the testing dependencies are still not present (e.g. all the various `pytest` plugins). Please add those somewhere with some documentation for how to set that up locally.

---

This file is used by the github actions sphinx documentation build. I would like to get rid of it, but I couldn't figure out how to get the build to install requirements from setup.py instead. I will add a `pip install -e .[test]` to the documentation. Or conda-build?